ansible-collection-redhatci-ocp.spec

```diff
@@ -51,6 +51,10 @@ find -type f ! -executable -name '*.py' -print -exec sed -i -e '1{\@^#!.*@d}' '{
 
 
 %changelog
+* Fri Jul 19 2024 Ramon Perez <raperez@redhat.com> - 0.13.EPOCH-VERS
```

Comment:
This date is fine, but if possible let's bump it up when ready to merge

Reply:
Bad thing is that the agent changes need to be pass Zuul check before merging, so I don't really know when I'm going to merge this :/ I'll keep an eye on this in the meantime

Reply:
done